src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -23,7 +23,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
                         ' It is not permissible to access the Left property of a BoundUserDefinedBinaryOperator unconditionally,
                         ' because that property can throw an exception if the operator expression is semantically invalid.
                         ' Fetching the left operand through IBinaryOperatorExpression is safe.
-                        If rightOperatorBinary.LeftOperand Is value.LeftOnTheRightOpt Then
+                        If rightOperatorBinary.LeftOperand Is Create(value.LeftOnTheRightOpt) Then
```

Comment:
>Create(value.LeftOnTheRightOpt) [](start = 62, length = 31)

I am not sure I understand the motivation behind this change. Could you please elaborate?